src/styles.css

```
@@ -25,7 +25,11 @@
 }
 html{scroll-behavior:smooth}
 body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-serif;min-height:100vh;overflow-x:hidden;transition:background .3s,color .3s}
+<<<<<<< HEAD
```

Stylelint (17.11.1)

[error] Expected "HEAD" to be "head" (selector-type-case)

(selector-type-case)